### ISSUE TITLE
Volumetric lighting

### ISSUE THREAD

**Issue by Worick18** [GTAmodding/GTAIV-Issues-List#60]
The Xbox 360 version has volumetric lighting in some rooms that interacts with objects. Presumably in the PC version, all volumetric lighting does not interact with anything.
https://www.youtube.com/watch?v=4j11ksebG7k&feature=emb_logo

**Comment by valsorai** [comment]
Yes I support this! Wanted to request this too because I would like to see this in the Fusion fix

**Comment by Undisputed00x** [comment]
There is a recent mod that i found out and it is awesome 
https://gtaforums.com/topic/989098-console-visuals-the-complete-edition
one of creators link https://twitter.com/Ash_735/status/1670932594567806978
